cpg_workflows/large_cohort/sample_qc.py

```diff
@@ -121,14 +122,15 @@ def impute_sex(
             logging.info(f'count post {name} filter:{vds.variant_data.count()}')
 
     # Infer sex (adds row fields: is_female, var_data_chr20_mean_dp, sex_karyotype)
+    inf_ploidy_using_var = get_config()['large_cohort'].get('inf_ploidy_using_var')
```

Comment:
This will return `None` if the config value isn't specified - do we want a default False here, or to raise an exception if the value isn't supplied?

Reply:
I think the general consensus in the thread was to have the default value of `False` though this is still dependent on how setting the variables to false effects results across whole cohorts